src/lib.rs

```diff
@@ -186,42 +179,87 @@ pub struct AxisId(u32);
 pub struct ButtonId(u32);
 
 /// Provides a way to retreive events from the windows that were registered to it.
-// TODO: document usage in multiple threads
-#[derive(Clone)]
+///
+/// To wake up an `EventsLoop` from a another thread, see the `EventsLoopProxy` docs.
 pub struct EventsLoop {
-    events_loop: Arc<platform::EventsLoop>,
+    events_loop: platform::EventsLoop,
+}
+
+/// Returned by the user callback given to the `EventsLoop::run_forever` method.
+///
+/// Indicates whether the `run_forever` method should continue or complete.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum ControlFlow {
+    /// Continue looping and waiting for events.
+    Continue,
+    /// Exit from the event loop.
+    Complete,
```

Comment:
Can this variant be `Break` instead of `Complete` to mirror the normal control flow keywords?